igvf_utils/scripts/eu_get_accessions.py

```diff
@@ -9,18 +9,18 @@
 ###
 
 """
-Use this script when there are records on the ENCODE Portal for which you know their aliases, but
+Use this script when there are records on the IGVF Portal for which you know their aliases, but
 want to retreive their DCC accessions. This will only work if the record aliases you provided are
```

Comment:
DCC is not a thing, so probably better `IGVF accessions` or  `portal accessions`